addons/crm/views/crm_lead_views.xml

```diff
@@ -576,47 +576,111 @@
-                    <field name="activity_summary" optional="hide" readonly="1"/>
-                    <field name="stage_id" required="1" optional="show"/>
-                    <field name="probability" avg="Avg. of Probability" optional="hide"/>
-                    <field name="user_id" optional="show"/>
+                    <field name="country_id" optional="hide"/>
+                    <field name="user_id" widget="many2one_avatar_user" optional="show"/>
                     <field name="team_id" optional="show"/>
-                    <field name="company_currency" invisible="1"/>
-                    <field name="planned_revenue" sum="Expected Revenues" optional="show" widget='monetary' options="{'currency_field': 'company_currency'}"/>
-                    <field name="referred" invisible="1"/>
-                    <field name="message_needaction" invisible="1"/>
-                    <field name="probability" invisible="1"/>
-                    <field name="write_date" invisible="1"/>
-                    <field name="company_id" groups="base.group_multi_company" optional="show" />
+                    <field name="priority" optional="hide"/>
+                    <field name="activity_ids" widget="list_activity"/>
+                    <field name="activity_user_id" optional="hide" string="Activity by" widget="many2one_avatar_user"/>
+                    <field name="activity_date_deadline_my" string="My Deadline" widget="remaining_days" options="{'allow_order': '1'}"/>
                     <field name="campaign_id" optional="hide"/>
                     <field name="medium_id" optional="hide"/>
                     <field name="source_id" optional="hide"/>
+                    <field name="company_currency" invisible="1"/>
+                    <field name="planned_revenue" sum="Expected Revenues" optional="show" widget="monetary" options="{'currency_field': 'company_currency'}"/>
+                    <field name="stage_id" optional="show" decoration-bf="1"/>
+                    <field name="active" invisible="1"/>
+                    <field name="probability" optional="hide"/>
+                    <field name="company_id" groups="base.group_multi_company" optional="show"/>
                     <field name="tag_ids" optional="hide" widget="many2many_tags" options="{'color_field': 'color'}"/>
-                    <field name="priority" optional="hide"/>
-                    <field name="activity_exception_decoration" widget="activity_exception"/>
+                    <field name="referred" invisible="1"/>
+                    <field name="message_needaction" invisible="1"/>
                 </tree>
             </field>
         </record>
 
+        <record id="crm_lead_view_list_activities" model="ir.ui.view">
+            <field name="name">crm.lead.list.activities</field>
+            <field name="model">crm.lead</field>
+            <field name="mode">primary</field>
+            <field name="priority">15</field>
```

Comment:
Priority 15 means this will be the default view.  I think this should be the inverse: setting 1 to opportunity view, 10 to lead view (as leads is an optional step that should not be the default view), and 20 to activity view, something like that.